src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -6,20 +6,20 @@
 
 namespace ePiggyWeb.DataManagement.Saving
 {
-    public static class AlternativeSavingCalculator
+    public class AlternativeSavingCalculator
```

Comment:
After answer from the lecturer I think we actually leave this class static, but the again what about if it uses threads? :D Or as I understand this one won't use it?

Reply:
Well the asynchronous part uses this call, so i made it non-static just in case, though I'm definitely willing to change it back, if that's the better option.